scripts/ci/run-test-interpreter.sh

```diff
@@ -14,7 +14,7 @@ ${TESTCMD} --label=System.Core --timeout=160m make -w -C mcs/class/System.Core r
 ${TESTCMD} --label=mcs-tests --timeout=160m make -w -C mcs/tests run-test V=1;
 ${TESTCMD} --label=Mono.Debugger.Soft --timeout=5m make -w -C mcs/class/Mono.Debugger.Soft run-test V=1
 
-if [[ ${CI_TAGS} != *'pull-request'* ]] || [[ ${CI_TAGS} != *'arm'* ]]; then
+if [[ ${CI_TAGS} != *'pull-request'* ]] && [[ ${CI_TAGS} != *'arm'* ]]; then
```

Comment:
@BrzVlad do you remember what the intend was back then? https://github.com/mono/mono/pull/9648/files#diff-5b6edc3ae02a8f8ef6e0cc12afca7eafR17

Reply:
I'm confused. The code looks ok to me. The idea was to not run full arm on PRs because it takes too long. (Comparing on master builds amd64 vs arm => 1:15 vs 3:30). Arm builds are already always lagging behind on PRs, this would make it worse.

And why would this PR make never running full arm tests ?

Reply:
Johan is leaving PRs alone, still "fast", but also disabling on master.
I'd'a'thought you'd want to test *sometimes* on master, like daily or weekly. ?

Reply:
OK, lets reset this to know what should be supported. Right now we run full test suite on PR's for all platforms except arm.

Doesn't look like we have any arm interpreter CI lanes on CI master.

So if the intent is to run full PR test suite on amd64 platforms then the above is not correct, but then this can't be done on Windows (takes to long time to run, similar to arm).

So if we still want to run full PR test suite on Linux and full test suite on master for all platforms, then this PR needs to be adjusted to exclude Windows on PR builds.

Reply:
https://jenkins.mono-project.com//job/test-mono-interpreter/

Reply:
Ahh, sorry didn't find that job when I search. I will adjust the PR to exclude Windows on PR builds similar to arm.

Reply:
maybe it would look better and clearer to add in the scripts stuff like : if (pr and arm) exit; if (windows) exit; ... then the running of all test suites. And add comment to small explanation of why we don't run so it is no confusion

Reply:
Agree, will make the condition clearer as well.

Reply:
I curious why not test Windows sometimes, like daily or weekly, not PRs?

Reply:
Full test suite will be run on daily master builds for Windows as well.